src/v0/destinations/af/deleteUsers.js

```diff
@@ -89,9 +89,9 @@ const userDeletionHandler = async (userAttributes, config) => {
           if (!body.property_id) {
             throw new ErrorBuilder()
               .setMessage(
-                "appleAppId is required for ios_advertising_id type identifier"
+                "[Appsflyer]::appleAppId is required for ios_advertising_id type identifier"
               )
-              .setStatus(500)
+              .setStatus(400)
```

Comment:
Shouldn't we treat configuration errors as retryable?

Reply:
Yes

Reply:
Fixed